src/test/benchmarks/jetton/results.json

```diff
@@ -54,11 +54,17 @@
       "burn": "16029",
       "discovery": "10093"
     },
+    {
+      "label": "1.5.3 with improved storeBool",
```

Comment:
let's add the link to this PR or some other way of identifying the exact version of Tact here (the commit hash will be known only after we squash and merge)

Reply:
Done